src/js/utils/dom.js

```diff
@@ -209,3 +209,11 @@ export function nextSibling(element) {
 export function previousSibling(element) {
     return element.previousElementSibling;
 }
+
+export function open(link, target, additionalOptions = {}) {
+    let a = document.createElement('a');
```

Comment:
let's write some tests!

Reply:
you should be able to mock `document.createElement` temporarily for the purpose of the test so you can assert the result of these changes.